install_config/upgrading/migrating_etcd.adoc

```diff
@@ -286,9 +277,7 @@ from a backup:
 . Stop the master services:
 +
 ----
-# systemctl stop atomic-openshift-master \
-    atomic-openshift-master-api \
-    atomic-openshift-master-controllers
+# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
```

Comment:
Here should be "stop", but not "restart", if I am right.

Reply:
@jianlinliu Thank you! Thank you! Copy/paste error.

Reply:
@jianlinliu Thank you! Thank you! Copy/paste error.